documentation/translating.rst

```diff
@@ -125,6 +126,7 @@ in production; others are works in progress.
 .. _github_uk: https://github.com/python/python-docs-uk
 .. _github_zh_cn: https://github.com/python/python-docs-zh-cn
 .. _github_zh_tw: https://github.com/python/python-docs-zh-tw
+.. _github_ru: https://github.com/MLGRussianXP/python-docs-ru
```

Comment:
Does this need migrating under `https://github.com/python/`?

Reply:
Not too soon, but if it takes off, yes.